### ISSUE TITLE
Post a dummy Github review when posting

### ISSUE THREAD

**Issue by ibrahima** [Reviewable/Reviewable#463]
Might solve what benidhamma was asking for here: https://gitter.im/Reviewable/Reviewable?at=58d0328b58ad4eb028350c2e

Basically, it could be useful to have the option to let Reviewable post a Github review with the last Reviewable status (eg. if everything is resolved, or if you put :lgtm: in the top level comment box, or perhaps just have a checkbox or dropdown in the publish button), with a message like "See Reviewable". Lately my team has been using the new "request reviewer" feature on Github, but I still prefer using Reviewable for the actual review, so now I have to dismiss their request or post a dummy comment. If I could do that when publishing from Reviewable, that would be great!

Obviously this is much lower priority than the Vue rewrite or other things, but I just wanted to throw the idea out there, especially when I saw someone else ask for something similar.

**Comment by pkaminski** [comment]
Yep, this definitely needs to get done.  I'm actually leaning towards posting _all_ Reviewable comments as GitHub reviews.  It's not yet clear to me how best to integrate GitHub's "requested reviewer" and "review disposition" systems with Reviewable -- whether with a kludge, or by adopting the system wholesale and possibly dropping backwards compatibility for Reviewable's own.

But as you say, currently the front-end is lying on the floor in pieces as I work towards Vue-ifying everything, so this will need to wait a bit.
